pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/download/DownloadParticipantListService.java

```diff
@@ -0,0 +1,151 @@
+package org.broadinstitute.dsm.service.download;
+
+import static org.broadinstitute.dsm.util.ElasticSearchUtil.DEFAULT_FROM;
+import static org.broadinstitute.dsm.util.ElasticSearchUtil.DSM;
+import static org.broadinstitute.dsm.util.ElasticSearchUtil.MAX_RESULT_SIZE;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipOutputStream;
+
+import com.netflix.servo.util.VisibleForTesting;
+import lombok.extern.slf4j.Slf4j;
+import org.broadinstitute.dsm.db.KitRequestShipping;
+import org.broadinstitute.dsm.exception.DsmInternalError;
+import org.broadinstitute.dsm.model.at.SampleQueue;
+import org.broadinstitute.dsm.model.elastic.export.tabular.DataDictionaryExporter;
+import org.broadinstitute.dsm.model.elastic.export.tabular.ModuleExportConfig;
+import org.broadinstitute.dsm.model.elastic.export.tabular.TabularParticipantExporter;
+import org.broadinstitute.dsm.model.elastic.export.tabular.TabularParticipantParser;
+import org.broadinstitute.dsm.model.elastic.search.UnparsedDeserializer;
+import org.broadinstitute.dsm.model.elastic.search.UnparsedESParticipantDto;
+import org.broadinstitute.dsm.model.filter.FilterFactory;
+import org.broadinstitute.dsm.model.filter.Filterable;
+import org.broadinstitute.dsm.model.participant.DownloadParticipantListParams;
+import org.broadinstitute.dsm.model.participant.ParticipantWrapperDto;
+import org.broadinstitute.dsm.model.participant.ParticipantWrapperResult;
+import spark.QueryParamsMap;
+import spark.Response;
+
+@Slf4j
+
+public class DownloadParticipantListService {
+    /**
+     * Fetches participant data from Elasticsearch based on the given filter and query parameters.
+     * returns a list of ParticipantWrapperDto objects
+     */
+    @VisibleForTesting
```

Comment:
This was resolved by mistake and my reply to it is lost! 
I meant to write that this can not be protected because the test class needs access to it and they are not in the same package. Please let me know if I am missing something, I would appreciate it

Reply:
I see.
Why are they not in the same package?